src/game/Tactical/Points.cc

```diff
@@ -1116,18 +1116,37 @@ static UINT8 MinAPsToPunch(SOLDIERTYPE const& s, GridNo gridno, bool const add_t
 		{
 			ap += GetAPsToChangeStance(&s, ANIM_CROUCH);
 		}
-		else if (s.sGridNo == gridno)
+		else
 		{
 			ap += GetAPsToChangeStance(&s, ANIM_STAND);
 		}
 	}
 
-	if (add_turning_cost && s.sGridNo == gridno)
+	if (add_turning_cost && s.sGridNo != gridno)
```

Comment:
s.sGridNo != gridno
is always true by this point.

Reply:
Will remove the second condition as it's not needed.